pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -161,6 +163,20 @@ let
       ln -s $lib $out/lib/wasm32-wasi
     done
   '';
+
```

Comment:
might be worth adding a comment here explaining what this does/why we add this file

Reply:
Some branding, some telemetry, it's a bit inconclusive.

Reply:
Added.